platform/iphone/SCsub

```diff
@@ -16,6 +16,9 @@ iphone_lib = [
     'in_app_store.mm',
     'icloud.mm',
     'ios.mm',
+    'gamepad_iphone.mm',
+    'video_iphone.mm',
+    'motion_iphone.mm',
```

Comment:
Not exactly married to these names...

Reply:
Why not just `gamepad.mm`, `video.mm` and `motion.mm`? Unless there's some technical concern you had.

Reply:
Because using the platform identifier as suffix both in filename and class name is the convention in Godot's platform code.